### PR TITLE
Added rse presentation on front page and about us page

### PR DESCRIPTION
**Pull request by twinkarma**
* Added mailing list button
* Made links on dark background darker to add contrast

Screenshot from 2021-03-18 11-58-49
Screenshot from 2021-03-18 11-59-08

### CODE REVIEW

**Comment by davidwyld** [comment]
Looks great @twinkarma !

File: _layouts/home.html [hunk 1]
```diff
@@ -24,31 +24,68 @@
...
+                    <div>
+                        <a class="sm-a" href="/assets/slides/about-rse-march-2021.pdf" target="_blank">Slides for the above presentation</a>
+                    </div>
+                </div>
+                <div class="col-md">
+                    <div class="social-media-inner container">
+                        <h2 style="color: white">How can RSES help?</h2>
```
Comment (davidwyld) [inline _layouts/home.html]: Should this be "RSEs" (lower case 's') as in the above section on the homepage?

Comment (twinkarma) [reply _layouts/home.html]: It's meant to refer to our group research software engineering sheffield but yes I think it's a bit confusing as there's a mention of RSEs above. Maybe I should change it to "How can RSE Sheffield help?" instead or do you think "How can RSEs help?" is fine @davidwilby ?

**Comment by willfurnass** [comment]
@twinkarma Looks good - thanks for this!

**Pull request merged**